src/index.ts

```diff
@@ -488,20 +541,26 @@ export class Bigtable {
     const adminOptions = Object.assign(
       {},
       baseOptions,
+      getUniverseDomainOptions(options, options.BigtableTableAdminClient),
       {
         servicePath:
           customEndpointBaseUrl ||
-          getDomain('bigtableadmin', options.BigtableClient),
+          getDomain('bigtableadmin', options, options.BigtableTableAdminClient),
       },
       options,
     );
     const instanceAdminOptions = Object.assign(
       {},
       baseOptions,
+      getUniverseDomainOptions(options, options.BigtableInstanceAdminClient),
```

Comment:
Without this, ie. without providing a universeDomain to the gapic client we see an error that says: `Error: The configured universe domain (googleapis.com) does not match the universe domain found in the credentials (apis-tpczero.goog). If you haven't configured the universe domain explicitly, googleapis.com is the default.`

Reply:
Note that this should be BigtableInstanceAdminClient as this is a setting for the instance admin client.